deploy/sudo-scripts/README

```diff
@@ -0,0 +1,65 @@
+All things you need to do to only require 'sudo' once (i.e. for the sys-admin)
+==============================================================================
+
+Copy the required scripts to  `/usr/local/bin`
+----------------------------------------------
+
+```bash
+sudo cp firesim/deploy/sudo-scripts /usr/local/bin
+sudo cp firesim/platform/xilinx_alveo_u250/scripts/* /usr/local/bin
+```
+
+Change permissions / add the  `firesim` group
+---------------------------------------------
+
+```bash
+sudo addgroup firesim
+sudo chmod 775 /usr/local/bin/firesim*
+sudo chgrp firesim /usr/local/bin/firesim*
+```
+
+Allow the  `firesim` group to run the pre-installed commands
+------------------------------------------------------------
+
+Enter/create the following file with `sudo`:
+
+```bash
+sudo vi /etc/sudoers.d/firesim```
+```
+
+Add the following lines:
+
+```bash
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-chmod-xdma-perm
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-load-xdma-module
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-load-xvsec-module
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-mount
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-remove-dev-shm
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-remove-xdma-module
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-unmount
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-xvsecctl-flash-fpga
+%firesim ALL=(ALL) NOPASSWD: /usr/local/bin/firesim-fpga-util.py
+```
+
+Verify that a non-sudo account can run the commands
+---------------------------------------------------
+
+Enter a non-sudo enabled account that has access to the `firesim` group.
+
+```bash
+sudo -l
+```
+
+Have PCIe devices be `sudo` enabled
```

Comment:
This step could be done automatically by the manager like the other steps?

This would just require on the AWS-side: copying a script with this stuff to the run-farm instance, and running the script with the EBDF as an arg (how to get the EBDF on AWS?)

Locally: Just install the script like the others and run w/ the BDF given by enumeratefpgas.